### ISSUE TITLE
I think the loss in the code id wrong, how can you explain about it?

### ISSUE THREAD

**Issue by moving-on** [wallarm/nascell-automl#12]
self.cross_entropy_loss = tf.nn.softmax_cross_entropy_with_logits(logits=self.logprobs[:, -1, :], labels=self.states)

Why you use softmax_cross_entropy_with_logits here， the first state is "[10.0, 128.0, 1.0, 1.0]*args.max_layers"，so does the labels. The final output of RNN contributes to the action, why you use softmax on the action?

**Comment by moving-on** [comment]
for example:
state=[10.0, 128.0, 1.0, 1.0,10.0, 128.0, 1.0, 1.0], the final output with softmax is [0.1,0.1,0.1,0.1,0.1,0.1,0.2,0.2], then the loss is:
-(10*log0.1+128*log0.1+1*log0.1+1*log0.1+10*log0.1+128*log0.1+1*log0.2+1*log0.2)
what does this mean?

**Comment by gcooq** [comment]
me too, I think each layer should use a softmax function, not the whole output with a single softmax function.
